apps/docs/content/guides/auth/social-login/auth-workos.mdx

```diff
@@ -1,54 +1,53 @@
 ---
 id: 'auth-workos'
-title: 'Login with WorkOS'
-description: 'Add WorkOS OAuth to your Supabase project'
+title: 'SSO and Social Login with WorkOS'
+description: 'Add WorkOS AuthKit to your Supabase project'
 ---
 
-To enable WorkOS Auth for your project, you need to set up WorkOS OAuth application and add the application credentials to your Supabase Dashboard.
+[WorkOS AuthKit](https://authkit.com) provides email/password authentication and social login by default. Once AuthKit is installed, you can enable SSO through a dashboard toggle.
 
-## Overview
+## Step 1. Create a WorkOS organization
 
-In this guide, we will cover how to use Supabase OAuth with WorkOS to implement Single-Sign-On(SSO).
-
-The procedure consists of five broad steps:
-
-- Create a new organization from your WorkOS Dashboard.
-- Obtain the `Client ID` from the Configuration tab and configure redirect URI.
-- Obtain the `WorkOS Secret` from the credentials tab.
-- Connect a WorkOS Supported Identity Provider
-- Add your WorkOS credentials into your Supabase project
-
-## Create a WorkOS organization
-
-Log in to the dashboard and hop over to the Organizations tab to create and organization
+Log in to the WorkOS dashboard and visit the Organizations tab to create an organization
 ![Create an Organization](/docs/img/guides/auth-workos/workos-create-organization.png)
 
-## Obtain the client ID and configure redirect URI
+## Step 2. Obtain your `Client ID` and `WORKOS_API_KEY` values
 
-Head over to the Configuration tab and configure the redirect URI.The redirect URI should look like `https://<project-ref>.supabase.co/auth/v1/callback`
-Note that this is distinct from the redirect URI referred to in the Supabase dashboard
+![Get your Environment's Client ID and Secret](/docs/img/guides/auth-workos/workos-dashboard-get-client-id-and-key.png)
 
-![Fetch Client ID and configure Redirect URI](/docs/img/guides/auth-workos/workos-clientid-redirect-uri.png)
+Visit the getting started page of the WorkOS dashboard, at [https://dashboard.workos.com/get-started](https://dashboard.workos.com/get-started), to obtain the following values from the Quickstart panel:
+* `WORKOS_CLIENT_ID`
+* `WORKOS_API_KEY`A
```

Comment:
```suggestion
* `WORKOS_API_KEY`
```

Reply:
Thanks, fixed!

Reply:
Thank you - I found and fixed this today, too.